### PR TITLE
Bety python functions

### PR DESCRIPTION
**Pull request by nickheyek**
Added functions for getting data with the BETYdb API as python data structures.
Uses the same method as the R OpenSci package for importing the API key from file.

### CODE REVIEW

**Comment by max-zilla** [comment]
first skimthrough, it's off to a good start. i will look closer in a bit, but two general things:
- you can remove my placeholder stuff below (get_cultivar, etc)
- I dont think we need "bety_" before all the function names, since the scripts will generally look like:
```
import terrautils.betydb

terrautils.betydb.betydb_query()
terrautils.betydb.betydb_submit_traits()
```

The pattern we use elsewhere would be:
```
terrautils.betydb.query()
terrautils.betydb.submit_traits()
```

Also I'm adding @jterstriep because his group just pushed a branch as well: https://github.com/terraref/terrautils/tree/gdal_functions
...they haven't made a pull request but these constitute the two pieces of code that we can synthesize.

**Comment by nickheyek** [comment]
Excuse my lack of experience with this, but I made the changes you mentioned, should I close this and create new pull request?

**Comment by max-zilla** [comment]
@nickheyek nope, we can keep appending to this one. Jeff might add to this one as well. We can merge once we have all changes ready to go - we need to review first.

**Comment by dlebauer** [comment]
@nickheyek no need to create a new pull request. if you push to the same branch, which it appears you did,  the PR gets updated.

**Comment by nickheyek** [comment]
Thanks!

File: terrautils/betydb.py [hunk 1]
```diff
@@ -5,28 +5,84 @@
...
 from osgeo import ogr
 
+# Returns key from ~/.betykey if it exists
+def get_bety_key():
+    keyfile_path = os.path.expanduser('~/.betykey')
+    if os.path.exists(keyfile_path):
+        keyfile = open(keyfile_path, "r")
+        return keyfile.readline().strip()
+    else:
+        logging.error("~/.betykey does not exist; use 'betykey' argument or set ~/.betykey")
```
Comment (max-zilla) [inline terrautils/betydb.py]: let's also check ```os.getenv("BETYDB_KEY", False)``` before throwing error

**Comment by dlebauer** [comment]
@jterstriep says this code has been merged into #3, so closing this one as duplicate

**Pull request closed without merging**